client/galaxy/scripts/layout/router.js

```diff
@@ -12,17 +13,27 @@ var Router = Backbone.Router.extend({
         this.options = options;
     },
 
+    executeUseRouter: function(url) {
```

Comment:
I am not sure about the naming here, what is a 'use router'?

Reply:
It refers to the `__use_router__` navigation definition we use, this function executes the navigation intended by those targets.